### PR TITLE
Removes slaughter demon heart from wizard academy

### PR DESCRIPTION
**Pull request by SomeguyManperson**
:cl:  
rscdel: Slaughter demon with no AI in wizard academy no longer drops bloodcrawl ability button
/:cl:

### CODE REVIEW

**Comment by boodaliboo** [comment]
didnt know that this could happen but yeah needed

**Comment by Marmio64** [comment]
guy who used to rush academy every round whose name i forgot, think it was chester something, get destroyed nerd.

File: _maps/RandomRuins/SpaceRuins/Academy.dmm [hunk 1]
```diff
@@ -1577,6 +1577,7 @@
 /mob/living/simple_animal/slaughter{
 	desc = "A radioactive being made of condensed fusion material";
 	name = "Radioactive Prat Demon"
```
Comment (patpol4) [inline _maps/RandomRuins/SpaceRuins/Academy.dmm]: You need a ; between vars

**Review by patpol4 — APPROVED** [review]
no gaming on my watch

**Pull request merged**